label_studio/labels_manager/api.py

```diff
@@ -27,6 +27,8 @@
     name='create',
     decorator=swagger_auto_schema(
         tags=['Labels'],
+        x_fern_sdk_group_name='labels',
```

Comment:
should these be `['projects', 'labels']`? I do see that the operation description mentions projects.

Reply:
`Labels` are created globally and then linked to the projects via `LabelLinkAPI`.
On the other hand, we never productionalized the entire workflow with labels and user-defined custom categories in Taxonomy (especially right now when it is under `legacy` parameter). I propose to hide this endpoint in SDK until we readdress ontology management etc.